sol/main.c

```diff
@@ -541,16 +578,20 @@ run_sol(struct net_config *net_conf, struct sol_config *sol_conf)
 	front_inc = sol_conf->pri_req_max_len + sol_conf->deq_burst_size;
```

Comment:
`front_inc` really should be `2 * sol_conf->pri_req_max_len + sol_conf->enq_burst_size`.
Reason:
1. `sol_conf->pri_req_max_len` packets may sit at the ring;
2. `sol_conf->pri_req_max_len` packet may sit at the actually queue;
3. `enqueue_reqs()` temporarily adds `sol_conf->enq_burst_size` more packets;
4. `sol_conf->deq_burst_size` does not count because `dequeue_reqs()` only reduces the number of packets, that is, it does not add.